crates/bevy_reflect/bevy_reflect_derive/src/lib.rs

```diff
@@ -201,8 +202,10 @@ pub fn derive_reflect(input: TokenStream) -> TokenStream {
     };
 
     TokenStream::from(quote! {
-        #reflect_impls
-        #from_reflect_impl
+        const _: () = {
+            #reflect_impls
+            #from_reflect_impl
+        };
```

Comment:
should we apply this treatment to all the macros defined in this file?

Reply:
Fair, I'll add it!